### PR TITLE
Add Molecule class, xyz2mol

### PR DESCRIPTION
**Pull request by guenp**
Adds a convenience class for creating and visualizing molecules.

- [ ] Add UTs for Molecule
- [ ] Add docstrings
- [ ] Update example notebook
- [ ] Add copyright notices

### CODE REVIEW

File: Python/qdk-chemistry/qdk_chemistry/molecule.py [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+        solver: str,
+        base_path: str = None,
+        **parameters
+    ):
+        """Create input deck and save to file
+
+        :param name: Name of the input deck
+        :type name: str
+        :param file_name: Output file name
+        :type file_name: str
```
Comment (dgwvfxn) [inline Python/qdk-chemistry/qdk_chemistry/molecule.py:102]: name and file_name here is a bit confusing to me. Does name refer to the job name? and file_name refer to the ".nw/in/inp" name?

File: Python/qdk-chemistry/qdk_chemistry/xyz2mol/util.py [hunk 2]
```diff
@@ -0,0 +1,296 @@
...
+atomic_valence[8] = [2,1,3]
+atomic_valence[9] = [1]
+atomic_valence[14] = [4]
+atomic_valence[15] = [5,3] #[5,4,3]
+atomic_valence[16] = [6,3,2] #[6,4,2]
+atomic_valence[17] = [1]
+atomic_valence[32] = [4]
+atomic_valence[35] = [1]
+atomic_valence[53] = [1]
+
```
Comment (dgwvfxn) [inline Python/qdk-chemistry/qdk_chemistry/xyz2mol/util.py:49]: what's the meaning of the atomic_valence element? for example, atomic_valence[5] = [3,4] or atomic_valence[8] = [2,1,3]?

File: Python/qdk-chemistry/qdk_chemistry/molecule.py [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+        mol._updated = True
+        display(mol.design_widget)
+        return mol
+
+    def update_design(self, add_hs: bool = True, num_confs: int = 10):
+        mol = self.design_widget.to_mol(add_hs=add_hs)
+        self.mol = mol
+        self.widget = JsmolWidget.from_mol(mol=mol, num_confs=num_confs)
+
+    def create_input(
```
Comment (dgwvfxn) [inline Python/qdk-chemistry/qdk_chemistry/molecule.py:89]: for solver job, e.g. qpe, fci, can they be generated using this create_input function? In the doc, we planned to create individual classmethods, e.g. to_nwchem, to_psi4, to_fci, to_qpe for various jobs.

File: Python/qdk-chemistry/qdk_chemistry/xyz2mol/util.py (hunk 2 above)
Comment (guenp) [reply Python/qdk-chemistry/qdk_chemistry/xyz2mol/util.py:49]: Note from convo with @dgwvfxn: Would it be possible to use this to fix the issue of hypervalent bonds in e.g. Femoco?

**Comment by guenp** [comment]
Closing this PR to move functionality to new repo.

**Pull request closed without merging**